exporters/otlp/otlplog/otlploggrpc/client.go

```diff
@@ -6,9 +6,10 @@ package otlploggrpc // import "go.opentelemetry.io/otel/exporters/otlp/otlplog/o
 import (
 	"context"
 	"errors"
-	"fmt"
 	"time"
 
+	"go.opentelemetry.io/otel/exporters/otlp/otlplog/otlploggrpc/internal"
```

Comment:
This doesn't look correctly formatted, the local imports should all be in their own block at the bottom.

I'm not sure why the goimports formatter didn't catch this and fail the `lint` CI job.

Regardless, this should be moved below next to the other `internal` imports added.